pkg/util/cert.go

```diff
@@ -41,8 +40,16 @@ const (
 	ValidityTenYears = 10 * ValidityOneYear
 )
 
+func EnsureKeyPair(pubKeyPath, privKeyPath string) error {
+	if _, err := getKeyPair(pubKeyPath, privKeyPath); err == nil {
```

Comment:
Should we have a log if the error is not nil? Is it relevant enough even though its regenerated afterwards?

Reply:
If getKeyPair is not successful, it will return all errors internally. I'd say it's not necessary.